src/main/java/org/broadinstitute/hellbender/utils/test/IntegrationTestSpec.java

```diff
@@ -180,11 +181,16 @@ public static void assertEqualTextFiles(final File resultFile, final File expect
         final List<String> actualLines = new XReadLines(resultFile).readLines().stream().filter(startsWithComment.negate()).collect(Collectors.toList());
         final List<String> expectedLines = new XReadLines(expectedFile).readLines().stream().filter(startsWithComment.negate()).collect(Collectors.toList());
 
-        Assert.assertEquals(actualLines.toString(), expectedLines.toString());
+        //For ease of debugging, we look at the lines first and only then check their counts
+        final int minLen = Math.min(actualLines.size(), expectedLines.size());
+        for (int i = 0; i < minLen; i++) {
+            Assert.assertEquals(actualLines.get(i).toString(), expectedLines.get(i).toString(), "Line number " + i + " (not counting comments)");
+        }
+        Assert.assertEquals (actualLines.size(), expectedLines.size(), "line counts");
```

Comment:
no, because sometimes files are just a bit different and this would fail saying that the number of lines is different, which is true but not helpful. What is helpful is seeing the actual differences